src/platform/EFR32/freertos_bluetooth.c

```diff
@@ -264,13 +229,14 @@ void vApplicationMallocFailedHook(void)
 
 void vApplicationStackOverflowHook(TaskHandle_t pxTask, char * pcTaskName)
 {
-    (void) pcTaskName;
+    //(void) pcTaskName;
     (void) pxTask;
 
     /* Run time stack overflow checking is performed if
     configCHECK_FOR_STACK_OVERFLOW is defined to 1 or 2.  This hook
     function is called if a stack overflow is detected. */
-
+    EFR32_LOG("TASK OVERFLOW");
```

Comment:
Doesn't this overflow the stack further and potentially clobber state needed to debug the failure?

Reply:
Well I did have to use it to debug a task overflow and it didn't overflow the stack further. If the stack is way too corrupt, we'll likely never reach this handler and fall directly into a Hard Fault.

Reply:
Ok, fair enough. I guess if you have a debugger attached you try again with a breakpoint at the beginning of this function anyway.